inc/classes/class-imagify.php

```diff
@@ -470,7 +470,7 @@ private function curl_http_call( $url, $args = [] ) {
 			$url = self::API_ENDPOINT . $url;
 			$ch  = @curl_init();
```

Comment:
Note: In PHP 8, `@` will no longer silence fatal errors. [See the notes here](https://php.watch/versions/8.0/fatal-error-suppression).

Reply:
As nearly as I  can tell, the error suppression here would have been to avoid a fatal error when `curl_init()` was not available. From v1.7.1, there is a separate check, `static supports_curl()` added to `Imagify_Requirements` class to throw an exception for that possibility, so the `@` suppression is redundant here in any version. Any other errors from `curl_init()` itself will cause it to return `false` which we're now going to check for (again, works in any version).